test/hydrated_bloc_delegate_test.dart

```diff
@@ -25,11 +27,9 @@ void main() {
       bloc = MockBloc();
     });
 
-    tearDown(() async {
-      final file = File('./.hydrated_bloc.json');
-      if (file.existsSync()) {
-        file.deleteSync();
-      }
+    tearDownAll(() async {
```

Comment:
Would it be possible to revert the `tearDownAll` change so that we can ensure the cache is wiped between each test run?

Reply:
Actually whole flutter test: +23 -4: Some tests failed.
Have an intention to postpone this merge a bit

Reply:
We can always merge it into my branch and I can work on updating the tests if you'd like. Just let me know 👍

Reply:
It's rather dev branch, so I think any work on tests would be awesome)
But I'm wondering could those test been broken by transition of `initialState` to `state`.
And is it a valid break

Reply:
it was a valid break and I fixed the tests on my branch. Please take a look when you have time, thanks!